src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -62,7 +70,29 @@ const CharacterForm = ({ onClose }) => {
     return params[0] === formModeEnum.edit
   }, [params])
 
-  const character = useSelector(characterSelectors.character(params[1]))
+  const {
+    error: getACharacterHasError,
+    loading: getACharacterIsLoading,
+    data: character,
```

Comment:
This should be something like 
```
    error: characterError,
    loading: isCharacterLoading,
    data: characterData,
```

I would also recommend renaming the query to `GET_CHARACTER`